templates/kv-explorer/metadata/snippets/component.txt

```diff
@@ -1,9 +1,12 @@
 [[trigger.http]]
-id = "trigger-kv-explorer"
-component = "kv-explorer"
+component = "{{project-name | kebab_case}}"
 route = "/internal/kv-explorer/..."
 
-[component.kv-explorer]
-source = { url = "https://github.com/fermyon/spin-kv-explorer/releases/download/v0.9.0/spin-kv-explorer.wasm", digest = "sha256:07f5f0b8514c14ae5830af0f21674fd28befee33cd7ca58bc0a68103829f2f9c" }
+[component.{{project-name | kebab_case}}]
+source = { url = "https://github.com/fermyon/spin-kv-explorer/releases/download/v0.10.0/spin-kv-explorer.wasm", digest = "sha256:07f5f0b8514c14ae5830af0f21674fd28befee33cd7ca58bc0a68103829f2f9c" }
```

Comment:
I updated this to the anticipated release number but the hash is therefore inaccurate. Should the template be updated in a subsequent PR or can we quickly follow this up with an update to the hash?

Reply:
As there will need to be an update anyway for the hash, it would be good to hold off the template until then, but it may not be worth the amount of picking apart of this PR!  You can also tag at the current HEAD so that Spin users (other than canary users) get the current template until you're ready to release, at which point you move the tag forward... but again, a bit of a faff if the release is going to be almost immediate!

Reply:
For not being tied to immediacy, i moved the template changes into a separate PR: https://github.com/fermyon/spin-kv-explorer/pull/28